From 617164b22dbbe17490377c56f8a859541e9fcfdb Mon Sep 17 00:00:00 2001
From: Bastien Nocera <hadess@hadess.net>
Date: Tue, 29 Oct 2013 11:34:20 +0100
Subject: [PATCH 1/3] Use strlen() instead of hard-coding string length

This avoids hard to detect bugs when we want a different string length,
and will be optimised by the compiler anyway.

https://bugzilla.gnome.org/show_bug.cgi?id=711063
---
 libdmapsharing/daap-share.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libdmapsharing/daap-share.c b/libdmapsharing/daap-share.c
index 6b719f5..66cdfe6 100644
--- a/libdmapsharing/daap-share.c
+++ b/libdmapsharing/daap-share.c
@@ -922,7 +922,7 @@ databases_items_xxx (DMAPShare * share,
 		const gchar *s;
 		gchar *content_range;
 
-		s = range_header + 6;	/* bytes= */
+		s = range_header + strlen ("bytes=");	/* bytes= */
 		offset = atoll (s);
 
 		content_range =
-- 
1.8.3.1

